### PR TITLE
Citadel 0.0.2

### PR DESCRIPTION
**Pull request by AaronDewes**
This version of Citadel mostly includes bug fixes, but also a better CLI.

More release notes at: https://github.com/runcitadel/core/releases/tag/untagged-efa9372da42fc2231ad9 (Only visible to members of the @runcitadel org).

I'll do some more testing and then merge it once I get approval from both @pwltr and @WilliamConnatser.

### CODE REVIEW

File: app/lib/composegenerator/v1/utils/networking.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def getFreePort(networkingFile: str, appId: str):
     # Ports used currently in Citadel
     # TODO: Update this list, currently it's outdated
     usedPorts = [80, 433, 443, 8333, 8332, 28332, 28333, 28334, 10009, 8080, 50001, 9050, 3002, 3000, 3300, 3001, 3004, 25441,
-                 3003, 3007, 3006, 3009, 3005, 8898, 3008, 8081, 8082, 8083, 8085, 2222, 8086, 8087, 8008, 8088, 8089, 8091]
+                 3003, 3007, 3006, 3009, 3010, 3005, 8898, 3008, 8081, 8082, 8083, 8085, 2222, 8086, 8087, 8008, 8088, 8089, 8091]
```
Comment (AaronDewes) [inline app/lib/composegenerator/v1/utils/networking.py]: @pwltr  Just FYI: I'll remove port 3010 soon, this list should only contain public ports. Port 3010 should not be exposed to the outside, only port 8000 (for nginx), which I'll add soon

**Comment by pwltr** [comment]
@AaronDewes There's 2 more PRs in manager and sdk. The manager one at least should probably go into this release too.

**Comment by AaronDewes** [comment]
I pushed a few cleanups to the ports list, but I won't finish that for this release, because it will take some time to test & ensure there are no apps which actually used ports in this list.

**Comment by pwltr** [comment]
Yeah don't worry about the ports, that was just to be able to do `citadel-dev.local:8000` for development for the new UI. There may   actually be a conflict, should really check that.

File: docker-compose.yml [hunk 2]
```diff
@@ -77,8 +77,8 @@ services:
     networks:
       default:
         ipv4_address: $BITCOIN_IP
-  lnd:
-    container_name: lnd
+  lightning:
```
Comment (pwltr) [inline docker-compose.yml:80]: @AaronDewes need to change the container name in some more places. lnd -> lightning
EDIT: fixed [911988f](https://github.com/runcitadel/core/pull/17/commits/911988ff37468b3eff4290ee2d1d42cdd690ed8f)

**Review by pwltr — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (AaronDewes) [reply docker-compose.yml:80]: Thanks!

**Review by WilliamConnatser — APPROVED** [review]

**Pull request merged**